README.md

```diff
@@ -32,7 +32,11 @@ at their own discretion.
     help projects adopt and iterate on policies and processes that achieve
     these goals and eventually allow them to graduate the incubation phase.
 
-      * Mikeal Rogers (@mikeal)
+      * Mikeal Rogers (@mikeal) Currently assigned to Express.
```

Comment:
Jeremiah isn't assigned as a mentor to Express, he's an active contributor there.